SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -124,6 +158,18 @@ public static FilterDefinition<CaseSubmission> GenerateFilter(QueryCaseSubmissio
                 filter &= MongoDB.Bson.Serialization.BsonSerializer.Deserialize<BsonDocument>(bsonQuery);
             }
 
+            if (addDeletedRecordsFilter)
+            {
+                if (request.IncludeDeletedRecords)
+                {
+                    filter &= (Builders<CaseSubmission>.Filter.Eq(s => s.Deleted, true) | Builders<CaseSubmission>.Filter.Ne(s => s.Deleted, true));
```

Comment:
Worth double checking but I think the way &= for filter works is "subtractive", if you want to have all records (deleted or not) you don't need to add anything to the filter, by default they should both be included.

Reply:
I had a look and that's true. I've updated this so the logic works that way. Nice!